hazelcast-client/src/main/java/com/hazelcast/client/spi/ClientPartitionService.java

```diff
@@ -37,10 +37,4 @@
 
     Partition getPartition(int partitionId);
 
-    /**
-     * @param partitionCount new partition count
-     * @return true if partitions are not checked yet or last partition count is equal to the partition count provided,
-     * otherwise return false
-     */
-    boolean isPartitionCountConsistent(int partitionCount);
```

Comment:
Is it backward compatible to delete from a public SPI ?

Reply:
This is introduced at 3.12 so it will be deleted before published. 
And we are not preserving SPI on the client.